packages/core/src/routes/connector-services.ts

```diff
@@ -0,0 +1,83 @@
+import { buildRawConnector } from '@logto/cli/lib/connector/index.js';
+import type { ConnectorFactory } from '@logto/cli/lib/connector/index.js';
+import {
+  type SmsConnector,
+  type EmailConnector,
+  demoConnectorIds,
+  VerificationCodeType,
+} from '@logto/connector-kit';
+import { phoneRegEx, emailRegEx } from '@logto/core-kit';
+import { arbitraryObjectGuard, ConnectorType } from '@logto/schemas';
+import { string, object } from 'zod';
+
+import RequestError from '#src/errors/RequestError/index.js';
+import koaGuard from '#src/middleware/koa-guard.js';
+import assertThat from '#src/utils/assert-that.js';
+import { loadConnectorFactories } from '#src/utils/connectors/index.js';
+
+import type { AuthedRouter, RouterInitArgs } from './types.js';
+
+export default function connectorServicesRoutes<T extends AuthedRouter>(
+  ...[router]: RouterInitArgs<T>
+) {
+  router.post(
```

Comment:
Also better avoid defining new terms. Do we have a shared convention of connector-services?

Reply:
Perhaps not. Since the previously mentioned connector test API is not an original CRUD operation on the `connector` DB and is more like a service.